respy/shared.py

```diff
@@ -353,7 +324,7 @@ def normalize_probabilities(probabilities):
 @nb.guvectorize(
     ["f8, f8, f8, f8, f8, f8[:], f8[:]"],
     "(), (), (), (), () -> (), ()",
-    nopython=True,
+    nopython=False,
```

Comment:
No, if you use `nopython=False`, this function is not better than any normal Python function. it must be True.